static/js/place/place.ts

```diff
@@ -123,6 +131,18 @@ async function getLandingPageData(
     });
 }
 
+/**
+ * Handler for NL search bar
+ * @param q search query entered by user
+ */
+function onSearch(q: string): void {
+  triggerGAEvent(GA_EVENT_NL_SEARCH, {
+    [GA_PARAM_QUERY]: q,
+    [GA_PARAM_SOURCE]: GA_VALUE_SEARCH_SOURCE_EXPLORE_LANDING,
```

Comment:
Should this source be the place page instead of explore landing?

Reply:
Good catch! Updated the source to be the place page.